public/api/conf/1.0/application.raml

```diff
@@ -115,13 +115,17 @@ traits:
       securedBy: [ sec.oauth_2_0: { scopes: [ "common-transit-convention-guarantee-balance" ] } ]
       responses:
         200:
+          description: "HTTP status 200 (OK) responses can be in the states of \"Pending request\", \"Successful request\", and \"Failed request\" as described below."
           body:
             application/json:
               type: !include schemas/get-balance-request-response-schema.json
               examples:
                 pending-request:
-                  description: Pending request
-                  (annotations.documentation): This is a pending request that has not yet received a response.
+                  description: "HTTP status 200 (OK) responses can be in the states of \"Pending request\", \"Successful request\", and \"Failed request\" as described below."
```

Comment:
This isn't ideal, but we have to do this in order to get this message to appear at the top of the examples section - the description under `200.description` doesn't appear anywhere unfortunately.